java/client/src/org/openqa/selenium/support/locators/RelativeLocator.java

```diff
@@ -87,14 +88,31 @@
       throw new UncheckedIOException(e);
     }
   }
+
   private static final int CLOSE_IN_PIXELS = 100;
 
   /**
    * Start of a relative locator, finding elements by tag name.
    */
-  public static RelativeBy withTagName(String tagName) {
+
+  public static RelativeBy with(By by) {
+    Require.nonNull("By to look for", by);
+    return new RelativeBy(by);
+  }
+
+  public static By tagName(String tagName) {
```

Comment:
These can all be safely removed now that `with(By)` exists. People can use a static import to get a short `tagName` method call to allow `with(tagName("foo"))`